apiFunctions/services.ts

```diff
@@ -29,8 +30,8 @@ export const getAllResources = async (filter = ''): Promise<Resource[]> => {
   return response.data
 }
 
-export const getServicesList = async (id: string): Promise<ServicesList> => {
-  const response = await ServicesAxiosClient.get<ServicesList>(
+export const getCollection = async (id: string): Promise<Collection> => {
+  const response = await ServicesAxiosClient.get<Collection>(
```

Comment:
There's def more stuff to rename but think it makes sense to chip away at it where we can